packages/pages/src/common/src/module/internal/_server.tsx

```diff
@@ -0,0 +1,21 @@
+// Environment: server
+
```

Comment:
Are these any different from the template ones? Doesn't look like it. We could just reuse those. But in reality we shouldn't put things on the page the user hasn't coded. All we really need is a div that mounts to the name of the module.

```
<div id="moduleName"><!--YEXT-SERVER--></div>
```

Reply:
They don't have PageProps like the template ones do, but otherwise they are the same.

Do the template clients/servers also have extra stuff we shouldn't have?

Reply:
We don't need the head tags stuff I don't think (unless it's necessary for the Vite css code to inject into). So if we don't need head then it'd just be like

```
<html>
  <div id="moduleName"><!--YEXT-SERVER--></div>
</html>
```

Reply:
I get an error No head tag is defined if I remove the head tags.

Going to keep this as is since it works and matches what we do for templates.